docs/getting-started/example-bpf-local.md

```diff
@@ -8,7 +8,7 @@ Assume the following command is run:
 
 ```console
 cd bpfman/examples/go-xdp-counter/
-sudo ./go-xdp-counter -iface eno3
+go run -exec sudo . -iface eno3
```

Comment:
I'm not a huge fan of this format, but I'm not a Go purest. There are 4 places in https://github.com/bpfman/bpfman/blob/main/docs/getting-started/example-bpf.md#running-examples that we show `sudo ./go-xxx-counter ...` examples. I guess we should keep them consistent.

Reply:
The reason for the `go run -exec sudo` idiom is that it
- works also for unit testing,
- allows quick turn-around for on-the-fly experiments,
- doesn't leave unwanted binaries lying around.

Reply:
Ack